apiserver/root.go

```diff
@@ -218,7 +218,7 @@ func BasicAuthCheck(username, password string) bool {
 
 	psw := string(secret.Data["password"])
 	if psw != password {
-		log.Errorf("%s  %s password does not match for user %s ", psw, password, username)
+		log.Errorf("password does not match for user %s ", username)
```

Comment:
remove whitespace at the end of the string.

Reply:
you beat me to this, I was just about to push that fix.